File: 백준/Silver/1929. 소수 구하기/소수 구하기.c
#include <stdio.h>

int is_prime(int num)
{
  int i;

  if (num == 1)
    return (0);
  i = 2;
  while (i*i <= num)
  {
    if (num % i == 0)
      return (0);
    i++;
  }
  return (1);
}

int main()
{
  int M, N;

  scanf("%d %d", &M, &N);
  for(int i = M; i <= N; i++)
  {
    if (is_prime(i) == 1)
      printf("%d\n", i);
  }
  return (0);
}

File: 백준/Silver/1929. 소수 구하기/소수 구하기.py
def is_prime(num):
    if num == 1:
        return False
    i = 2
    while i * i <= num:
        if num % i == 0:
            return False
        i += 1
    return True


def sol():
    M, N = tuple(map(int, input().split()))
    for i in range(M, N + 1):
        if is_prime(i) == True:
            print(i)


sol()
